AFv1/custom-domain-names.md

```diff
@@ -27,13 +27,13 @@ This document is for users of AppFog v1. This document does not apply to the cur
 <p>Alternatively, you can use the CLI to map the custom domain to your application -- follow the instructions below.</p>
 <h3 id="custom-domain-af-cli">Command-line Tool</h3>
 <ol>
-<li>If you haven't already, install the <a href="http://docs.appfog.com/getting-started/af-cli">af command line tool</a> and log in.</li>
+<li>If you haven't already, install the <a href="https://www.ctl.io/knowledge-base/afv1/appfog-cli-tool-manual/">af command line tool</a> and log in.</li>
 <li>Map the domain. $<code>af map</code></li>
 </ol>
 <p>For example:</p>
 <pre>$ af map {appname} www.example.com</pre>
 <h3 id="custom-domain-dns">Update your DNS Records</h3>
-<blockquote><strong>Note: </strong>If you intend to use SSL with your site, <strong><em>STOP HERE</em></strong> and go check out our <a href="http://docs.appfog.com/customize/ssl">SSL for Custom Domains</a> documentation. It includes information on the different DNS configuration needed for SSL Endpoint termination.</blockquote>
+<blockquote><strong>Note: </strong>If you intend to use SSL with your site, <strong><em>STOP HERE</em></strong> and go check out our <a href="https://www.ctl.io/knowledge-base/afv1/configuring-ssl-for-a-custom-domain/">SSL for Custom Domains</a> documentation. It includes information on the different DNS configuration needed for SSL Endpoint termination.</blockquote>
```

Comment:
This should be a relative link.

https://github.com/CenturyLinkCloud/PublicKB#links-kb-article-to-kb-article